### ISSUE TITLE
DefaultValue TCK test has incorrect default CAPE payload

### ISSUE THREAD

**Issue by tmiddlet2666** [eclipse/microprofile-graphql#208]
In the DefaultValue spec and TCK at https://download.eclipse.org/microprofile/microprofile-graphql-1.0.1/microprofile-graphql.html#default_values it has the following static for CAPE:

```
public final static String CAPE =
        "{" +
        "   \"id\": 1000," +
        "   \"name\": \"Cape\","+
        "   \"powerLevel\": 3," +
        "   \"height\": 1.2," +
        "   \"weight\": 0.3," +
        "   \"supernatural\": false" +
        "}";
```
See https://github.com/eclipse/microprofile-graphql/blob/1.0.1/tck/src/main/java/org/eclipse/microprofile/graphql/tck/apps/superhero/model/Item.java#L53

I see my own tck test fail as it cannot parse the generated mutation provisionHero as below.

```
provisionHero(
hero: String,
item: ItemInput = {   "id": 1000,   "name": "Cape",   "powerLevel": 3,   "height": 1.2,  
 "weight": 0.3,   "supernatural": false,   
"dateCreated": "19 February 1900 at 12:00 in Africa/Johannesburg",  
 "dateLastUsed": "29 Jan 2020 at 09:45 in zone +0200"}
): SuperHero
```
The above CAPE static should not be pure JSON but should generate something like the following without quotes around the variables.
```
item: ItemInput = {   id: 1000,   name: "Cape",   powerLevel: 3,   height: 1.2,   
weight: 0.3,   supernatural: false,   
dateCreated: "19 February 1900 at 12:00 in Africa/Johannesburg",   
dateLastUsed: "29 Jan 2020 at 09:45 in zone +0200"}
```
This should be validated in a test.

This may also make the smallrye tck implementation tests fail.

**Comment by phillip-kruger** [comment]
Awesome ! Thanks @tmiddlet2666 . If you are keen you can add a test for this ?

**Comment by phillip-kruger** [comment]
@kaqqao Can you please comment ?

**Comment by andymc12** [comment]
It sounds like this might be an implementation issue.  I agree that we need a test case for this scenario, but the idea of using JSON as the string for the default value in the Java code was intentional.

IIUC, the implementation is having problems converting the JSON into the GraphQL SDL in the schema.  I think that a new test would flush out the implementation issue.

**Comment by tmiddlet2666** [comment]
@andymc12 I see what you mean. So to be clear the JSON is correct but the default for the input should be 
```
provisionHero(hero: String,
item: ItemInput = {   id: 1000,   name: "Cape",   powerLevel: 3,   height: 1.2,   
weight: 0.3,   supernatural: false,   dateCreated: "19 February 1900 at 12:00 in Africa/Johannesburg", 
dateLastUsed: "29 Jan 2020 at 09:45 in zone +0200"}
```

**Comment by andymc12** [comment]
Yes, this looks right to me.  Basically there is an implicit requirement on the implementation to convert the JSON in the `@DefaultValue` annotation into SDL for the schema.  You're pull request looks good - I will go ahead and merge.  Thanks for the help!

**Comment by phillip-kruger** [comment]
@tmiddlet2666 Can you do a PR against 1.0.x then we can include this in the 1.0.2 release ?
It's already merged into master.

Thanks :)

**Comment by phillip-kruger** [comment]
Fixed in #228
